CONTRIBUTING.rst

```diff
@@ -14,17 +14,15 @@ check `easy issues <https://github.com/nexB/scancode-toolkit/labels/easy>`_
 When contributing to ScanCode (such as code, bugs, documentation, etc.) you
 agree to the Developer `Certificate of Origin <http://developercertificate.org/>`_
 and the ScanCode license (see the `NOTICE <https://github.com/nexB/scancode-toolkit/blob/develop/NOTICE>`_ file).
-The same approach is used by the Linux Kernel developers and several other projects.
+The same approach is used by Linux Kernel developers and several other projects.
 
 For commits, it is best to simply add a line like this to your commit message,
 with your name and email::
 
     Signed-off-by: Jane Doe <developer@example.com>
 
 Please try to write a good commit message, see `good commit message wiki
-<https://aboutcode.readthedocs.io/en/latest/contributing/writing_good_commit_messages.html>` for
-details. In particular use the imperative for your commit subject: think that
-you are giving an order to the codebase to update itself.
+<https://aboutcode.readthedocs.io/en/latest/contributing/writing_good_commit_messages.html>`_ for details. In particular use the imperative for your commit subject: think that you are giving an order to the codebase to update itself.
```

Comment:
why put this on a single line instead of three?

Reply:
Formatting issue. Fixed!